notebooks/06-Custom-Extensions.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<p>should references to other chapters have links?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>How do you create links to other tutorials?</p>